### DISCUSSION TITLE
Please consider adding my profile | anime theme

### DISCUSSION THREAD

**Discussion by junioralive** [abhisheknaiidu/awesome-github-profile-readme#1144] [Show and tell]
Hello,

I'd like to suggest adding my anime-themed GitHub repo https://github.com/junioralive/junioralive to your collection. I think it's a great fit and would appreciate your consideration.

Thanks

**Comment by UltraInstinct0x** [comment]
Actually I love how it looks!

**Reply by junioralive** [reply]
Thanks bruh <3
